package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "test": "npm --loglevel=silent run test.lint && npm --loglevel=silent run test.unit",
     "start": "API_URL=http://localhost PORT=3000 API_PORT=4000 PGADMINPASSWORD=postgres PGPASSWORD=give_me_time GOOGLE_AUTH_MOCK=true docker-compose -f docker-compose.yml -f docker-compose.dev.yml up",
+    "start_with_google": "API_URL=http://localhost PORT=3000 API_PORT=4000 PGADMINPASSWORD=postgres PGPASSWORD=give_me_time GOOGLE_CLIENT_ID=673157831962-gcgp4mj9mgadau0nh9pbaikhbmqkl04d.apps.googleusercontent.com GOOGLE_CLIENT_SECRET=MXCrbpJhjMcgmzcNeIsCMKn_ GOOGLE_REDIRECT_URL=http://localhost:3000 docker-compose -f docker-compose.yml -f docker-compose.dev.yml up",
```

Comment:
just don't commit it and add an example with fake secret and id in the readme
PS: i've revoked this client id and secret